sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -173,7 +173,7 @@ internal static string GetOperationName(Activity activity, ref AzMonList MappedT
             return activity.DisplayName;
         }
 
-        internal static string GetV2OperationName(Activity activity, string? url,  ref AzMonList MappedTags)
+        internal static string GetNewSchemaOperationName(Activity activity, string? url,  ref AzMonList MappedTags)
```

Comment:
Todo: make this method return null instead of passing activity as parameter.

Reply:
Makes sense.

Reply:
This makes unit test very complex, OperationName needs validation at both `GetNewSchemaOperationName` and its parent.